db/migrate/20210413011944_create_invoices.rb

```diff
@@ -0,0 +1,10 @@
+class CreateInvoices < ActiveRecord::Migration[5.2]
+  def change
+    create_table :invoices do |t|
+      t.references :customer, foreign_key: true
+      t.integer :status
```

Comment:
this isn't best practice, but it works. we can decide as a team if we want to keep it or change it.